week-1/3-semantic-html/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>3. Semantic HTML - HTML, CSS and Git Exercises</title>
+    <title>3.</title>
```

Comment:
Make sure you review your changes using `git diff` before committing, so unexpected changes (like this title change) don't slip in :slightly_smiling_face: .

Reply:
okay